pfcpiface/main.go

```diff
@@ -19,13 +19,20 @@ const (
 	modeSim = "sim"
 )
 
+//grpc channel state
+const (
+	Ready = 2 //grpc channel state Ready
+)
+
```

Comment:
Internal details. Should not be in main.go

Reply:
done